modcc/symdiff.cpp

```diff
@@ -79,7 +79,7 @@ class FindIdentifierVisitor: public Visitor {
     void visit(IfExpression* e) override {
         if (!found()) e->condition()->accept(this);
```

Comment:
Is there any way `condition()` or `true_branch()` could ever be null as well?

Reply:
Semantically? No, these represent `if (cond) {then} {else}`, where only the `else` part is optional. If we technically can produce such an expression, I do not know, but it'd constitute a bug in the transformations. A user can never, though.

Reply:
Ok, makes sense.